README.md

```diff
@@ -24,8 +24,8 @@ which functions as the data backbone for prominent hedge funds, banks, and finan
 
 | Language      | Server Application | Client Application (OpenAPI) |
```

Comment:
We probably shouldn't have any references to `OpenAPI` anywhere... we don't really call it that.